### DISCUSSION TITLE
Password reset email contains a bad link

### DISCUSSION THREAD

**Discussion by rajid** [requarks/wiki#5771] [Error / Bug Report] [answered]
My wiki.js server is setup running on port 3443 (with an SSL cert which is not self-signed).  If a user clicks on the "Forgot Password" link, they receive an email message with a link to a page where they can reset their password.  Unfortunately, this link does NOT have the port 3443 in it, so it does not work!  I found that simply appending ":3443" to the hostname part of the link results in it working fine, there is an easy work-around.  It looks as though the code which sends the automated email doesn't take into account that the wiki may be running on something other than port 80.

Looks as if server/models/users.js needs to take into consideration "WIKI.config.port" where it's only using "WIKI.config.host".

**Comment by Phil-hostwiki** [comment, marked as answer]
Did  you add the port to the Site URL field in the admin area?
<img width="719" alt="Screenshot 2022-10-20 at 16 06 09" src="https://user-images.githubusercontent.com/82508256/196986814-b43c9bf0-be47-4e58-a77e-22d4cde9e0be.png">

**Comment by rajid** [comment]
Yes, that does it!  Thanks!  It seems strange that having configured the port in "config.yml" and having that port number be used internally for URL references between pages, I would still need to duplicate that information someplace else in order to have password reset links work correctly.  But...  I'm ok with it, since it works now!  Thanks a lot!
